### PR TITLE
Add a new <CountrySelect> component

### PR DESCRIPTION
**Pull request by yuki24**
This adds a new `<CountrySelect>` component that's going to be used in the new Buy Now flow: https://zpl.io/ad7yyk7

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 6e8b62fd94d7671aa3e1620378fb35b58cb21142

https://deploy-preview-1073--artsy-reaction.netlify.com

File: src/Styleguide/Components/CountrySelect.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+export interface CountrySelectProps extends PositionProps, SpaceProps {
+  selected?: string
+  onSelect?: (value) => void
+}
+
+export const CountrySelect = (props: CountrySelectProps) => {
+  return <LargeSelect {...props} options={CountrySelectOptions} />
+}
+
+const CountrySelectOptions = [
```
Comment (damassi) [inline src/Styleguide/Components/CountrySelect.tsx]: maybe `countrySelectOptions` or `COUNTRY_SELECT_OPTIONS`? TitleCase is typically reserved for classes / components and static data typically follows a conventional const format.

**Comment by damassi** [comment]
One tiny comment about naming but otherwise LGTM 👍

File: src/Styleguide/Components/CountrySelect.tsx (hunk 1 above)
Comment (yuki24) [reply src/Styleguide/Components/CountrySelect.tsx]: fixed

**Pull request merged**